cartridges/bm_algolia/cartridge/controllers/AlgoliaBM.js

```diff
@@ -45,6 +45,7 @@ function handleSettings() {
         algoliaData.setPreference('InStockThreshold', params.InStockThreshold.value * 1);
         algoliaData.setPreference('SearchApiKey', params.SearchApiKey.value);
         algoliaData.setPreference('AdminApiKey', params.AdminApiKey.value);
+        algoliaData.setPreference('IndexIDPrefix', params.IndexIDPrefix.value);
```

Comment:
What does "ID" refers too? I would simply name it `IndexPrefix`

Reply:
Renamed site preference.